test/standard-bridge/SettlementGatewayTest.sol

```diff
@@ -4,31 +4,12 @@ pragma solidity ^0.8.15;
 import "forge-std/Test.sol";
 import "../../contracts/standard-bridge/SettlementGateway.sol";
 import "../../contracts/interfaces/IWhitelist.sol";
-
-contract MockWhitelist is IWhitelist {
```

Comment:
We no longer need to mock whitelist contract. We can test interactions directly with unit tests